src/style/mixins.scss

```diff
@@ -63,6 +63,15 @@
   }
 }
 
+@mixin iui-text-selection($status: primary) {
+  &::selection,
```

Comment:
Whats the difference between these 2 selectors?

Reply:
`&` will apply to the same level where the mixin is used, `*` will apply to all descendants in the tree